ioannnikulin/minishell
# 68_tokenisation

CHECK AND MERGE 70_refactoring_insert_map https://github.com/ioannnikulin/minishell/pull/84 FIRST.
THEN CHECK AND MERGE 68 refactoring parsing #85.

In this PR i handled memory leaks up to parsing_command_test.c (its from where i run the parse_command function)
Some your comments here are not handled yet. And i think i pushed rule make pretestfclean in Makefile. If you are ok with it.

To verify parsing you can
make all
make test
./tests/test

(to see tests - check parsing_command_test.c :) )
